server/datastore/mysql/migrations/tables/20230315104937_EnsureUniformCollation.go

```diff
@@ -15,6 +15,12 @@ func init() {
 }
 
 func fixupSoftware(tx *sql.Tx, collation string) error {
+	// the query below can be expensive, increasing the sort buffer size
+	// for the session avoids errors when reading the rows.
+	_, err := tx.Exec("SET SESSION sort_buffer_size = 2560000000")
+	if err != nil {
+		return fmt.Errorf("increasing global sort buffer size: %w", err)
+	}
```

Comment:
I was getting:

> Error 1038 (HY001): Out of sort memory, consider increasing server sort buffer size

on the newly added check for `rows.Err` below. I figured this is the simplest way to go about this without touching this tricky migration too much, but I'm open to any ideas.